packages/svelte/src/internal/client/runtime.js

```diff
@@ -597,35 +572,38 @@ function recursively_collect_effects(
  * @param {import('./types.js').Effect} effect
  * @param {number} filter_flags
  * @param {boolean} [shallow]
- * @returns {import('./types.js').Effect[]}
+ * @returns {void}
  */
-function get_nested_effects(effect, filter_flags, shallow = false) {
-	/**
-	 * @type {import("./types.js").Effect[]}
-	 */
-	var render_effects = [];
+function flush_nested_effects(effect, filter_flags, shallow = false) {
 	/**
 	 * @type {import("./types.js").Effect[]}
 	 */
 	var user_effects = [];
-	// When working with custom-elements, the root effects might not have a root
-	if (effect.effects === null && (effect.f & BRANCH_EFFECT) === 0) {
-		return [effect];
+
+	var previously_flushing_effect = is_flushing_effect;
+	is_flushing_effect = true;
+
+	try {
+		// When working with custom elements, the root effects might not have a root
```

Comment:
what does 'the root effects might not have a root' mean?

Reply:
This was prior to this PR. The render effect is custom element logic has no root